modules/navier_stokes/src/actions/NSFVAction.C

```diff
@@ -0,0 +1,2289 @@
+                                    1.0,
+                                    "energy_scaling > 0.0",
+                                    "The scaling factor for the energy variables.");
+  params.addRangeCheckedParam<Real>("passive_scalar_scaling",
+                                    1.0,
+                                    "passive_scalar_scaling > 0.0",
+                                    "The scaling factor for the passive scalar field variables.");
+
+  params.addParamNamesToGroup(
+      "momentum_advection_interpolation energy_advection_interpolation "
+      "passive_scalar_advection_interpolation mass_advection_interpolation "
+      "momentum_face_interpolation energy_face_interpolation passive_scalar_face_interpolation "
+      "pressure_face_interpolation momentum_two_term_bc_expansion "
+      "energy_two_term_bc_expansion passive_scalar_two_term_bc_expansion "
+      "pressure_two_term_bc_expansion momentum_scaling energy_scaling "
+      "mass_scaling passive_scalar_scaling",
+      "Numerical control");
+
+  /**
+   * Parameter controlling the turbulence handling used for the equations.
+   */
+  params.addParam<std::vector<BoundaryName>>(
+      "mixing_length_walls",
+      std::vector<BoundaryName>(),
+      "Walls where the mixing length model should be utilized.");
+
+  ExecFlagEnum exec_enum = MooseUtils::getDefaultExecFlagEnum();
+  params.addParam<ExecFlagEnum>("mixing_length_aux_execute_on",
+                                exec_enum,
+                                "When the mixing length aux kernels should be executed.");
+
+  params.addRangeCheckedParam<Real>("von_karman_const",
+                                    0.41,
+                                    "von_karman_const > 0.0",
+                                    "Von Karman parameter for the mixing length model");
+  params.addRangeCheckedParam<Real>(
+      "von_karman_const_0", 0.09, "von_karman_const_0 > 0.0", "'Escudier' model parameter");
+  params.addRangeCheckedParam<Real>(
+      "mixing_length_delta",
+      1.0,
+      "mixing_length_delta > 0.0",
+      "Tunable parameter related to the thickness of the boundary layer."
+      "When it is not specified, Prandtl's original mixing length model is retrieved.");
+
+  params.addParamNamesToGroup("mixing_length_walls mixing_length_aux_execute_on von_karman_const "
+                              "von_karman_const_0 mixing_length_delta",
+                              "Turbulence");
+
+  // Create input parameter groups
+
+  params.addParamNamesToGroup(
+      "dynamic_viscosity density thermal_expansion thermal_conductivity specific_heat",
+      "Material property names");
+
+  params.addParamNamesToGroup(
+      "inlet_boundaries momentum_inlet_types momentum_inlet_function energy_inlet_types "
+      "energy_inlet_function wall_boundaries momentum_wall_types energy_wall_types "
+      "energy_wall_function outlet_boundaries momentum_outlet_types pressure_function",
+      "Boundary control");
```

Comment:
```suggestion
      "Boundary conditions parameters");
```

Reply:
I changed it to "Boundary condition parameters".